.github/workflows/swiftwasm.yml

```diff
@@ -63,6 +78,6 @@ jobs:
           version: ${{ env.WASMTIME_VERSION }}
       - run: swift --version
       - run: wasmtime -V
-      - run: swift sdk install $SWIFT_SDK_URL --checksum $SWIFT_SDK_CHECKSUM
-      - run: swift build -c release --build-tests --swift-sdk $TARGET_TRIPLE -Xlinker -z -Xlinker stack-size=$STACK_SIZE
-      - run: wasmtime --dir / --wasm max-wasm-stack=$STACK_SIZE .build/release/swift-formatPackageTests.xctest
+      - run: swift sdk install $THRAEDS_SWIFT_SDK_URL --checksum $THRAEDS_SWIFT_SDK_CHECKSUM
+      - run: swift build -c release --build-tests --swift-sdk $THREADS_TARGET_TRIPLE -Xlinker -z -Xlinker stack-size=$STACK_SIZE
+      - run: wasmtime --dir / --wasm max-wasm-stack=$STACK_SIZE --wasi threads .build/release/swift-formatPackageTests.xctest
```

Comment:
Should I leave `wasm32-unknown-wasi` test?

For example, when #214, I had to test the fix on both `wasm32-unknown-wasi` and `wasm32-unknown-wasip1-threads`.